client/lib/upgrades/actions/domain-management.js

```diff
@@ -296,6 +296,12 @@ function applyDnsTemplate( domainName, template, variables, onComplete ) {
 	} );
 }
 
+function getDnsTemplateConflicts( domainName, template, variables, onComplete ) {
```

Comment:
So, this is tricky. For new code, I'm guessing we should be adding new stuff here, instead of opting for doing it all the Redux Way (tm). So, adding an action creator, selectors, etc.
Though, in this case, I'm not sure that's worth the overhead?
Maybe just use `wpcom.getDnsTemplateConflicts` in your code directly? Example: https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/upgrades/domain-management/transfer/transfer-to-other-user/index.jsx#L92 ([which may or may not be a good idea](https://github.com/Automattic/wp-calypso/pull/14029#discussion_r117297285))